src/library-authoring/LibraryAuthoringPage.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 .library-authoring-sidebar {
-  min-width: 300px;
+  min-width: 320px;
```

Comment:
If header text is smaller than 320px, details tab goes under `More` dropdown option, hence increasing the min width to 320px.

Reply:
This might be fixed by https://github.com/openedx/frontend-app-authoring/pull/1293.. might need to double check if that merges before this one.